mkdocs.yml

```diff
@@ -315,12 +315,12 @@ plugins:
         "models/extras/objectmetadata.md": "user-guide/platform-functionality/objectmetadata.md"
         "models/extras/savedview.md": "user-guide/platform-functionality/savedview.md"
         "models/extras/staticgroupassociation.md": "user-guide/platform-functionality/staticgroupassociation.md"
-        "models/wireless/accesspointgroup.md": "user-guide/core-data-model/wireless/accesspointgroup.md"
         "models/wireless/supporteddatarate.md": "user-guide/core-data-model/wireless/supporteddatarate.md"
         "models/wireless/radioprofile.md": "user-guide/core-data-model/wireless/radioprofile.md"
         "models/wireless/wirelessnetwork.md": "user-guide/core-data-model/wireless/wirelessnetwork.md"
-        "models/wireless/accesspointgroupradioprofileassignment.md": "user-guide/core-data-model/wireless/accesspointgroup.md"
-        "models/wireless/accesspointgroupwirelessnetworkassignment.md": "user-guide/core-data-model/wireless/accesspointgroup.md"
+        "user-guide/administration/installation/docker.md": "user-guide/administration/installation-extras/docker.md"
+        "user-guide/administration/installation/health-checks.md": "user-guide/administration/installation-extras/health-checks.md"
+        "user-guide/administration/installation/selinux-troubleshooting.md": "user-guide/administration/installation-extras/selinux-troubleshooting.md"
```

Comment:
Bad merge? These are duplicates of lines 327-329 aren't they?

Reply:
Ah, let me fix that.